pkg/store/cache/memcached.go

```diff
@@ -53,15 +53,15 @@ func NewMemcachedIndexCache(logger log.Logger, memcached cacheutil.MemcachedClie
 	c.hits.WithLabelValues(cacheTypePostings)
 	c.hits.WithLabelValues(cacheTypeSeries)
 
-	level.Info(logger).Log("msg", "created memcached index cache")
+	level.Info(logger).Log("msg", "created cacheClient index cache")
```

Comment:
Nitpick: seems to be excessive renaming, maybe just `created index cache`?

Reply:
Thank you very much. Done by #4931.